python/usb_console/data_consumers.py

```diff
@@ -94,3 +95,6 @@ def save(self):
         filepath = os.path.join(self.data_dir, filename)
         with open(filepath, 'w') as logfile:
             logfile.write(self.log)
+    
+    def put(self, logline):
+        self.queue.put("("+str(datetime.now())+")"+"\n"+logline+"\n")
```

Comment:
Because of this change the unit test for this class now fails (you can see this in the Travis CI build). I forgot that changing this would break it. Could you fix it?